nxviz/plots.py

```diff
@@ -99,6 +105,8 @@ def __init__(
         node_label_color=False,
         group_label_position=None,
         group_label_color=False,
+        fontsize=10,
+        fontfamily = "serif", # Valid inputs ('serif', 'sans-serif', 'fantasy', 'monospace')
```

Comment:
I'd suggest putting the comment in the docstring and removing it from here.

```suggestion
        fontfamily = "serif",
```